services/rest/src/main/java/mil/nga/giat/geowave/service/rest/RestServer.java

```diff
@@ -202,4 +216,85 @@ public String toString() {
 			return "The route exists, but the command does not extend ServerResource";
 		}
 	}
+
+	/**
+	 * ServerResource to handle uploading files. Uses restlet fileupload.
+	 */
+	public class FileUploadServerResource extends
```

Comment:
Annotate this class with @GeowaveOperation, so that the RestServer startup code can find it.